src/v1/list_joined_rooms.rs

```diff
@@ -0,0 +1,44 @@
+//! [GET /_synapse/admin/v1/users/:user_id/joined_rooms](https://github.com/matrix-org/synapse/blob/master/docs/admin_api/user_admin_api.rst#list-room-memberships-of-an-user)
+
+use js_int::UInt;
+
+use ruma::{api::ruma_api, identifiers::UserId};
+
+ruma_api! {
+    metadata: {
+        description: "list room memberships of a user",
+        method: GET,
+        name: "query_user",
+        path: "/_synapse/admin/v1/users/:user_id/joined_rooms",
+        rate_limited: false,
+        authentication: AccessToken,
+    }
+
+    request: {
+        /// User ID
+        #[ruma_api(path)]
+        pub user_id: &'a UserId,
+    }
+
+    response: {
+        /// List all joined roo IDs.
+        pub joined_rooms: Vec<ruma::RoomId>,
```

Comment:
Could you import `RoomId` at the top instead of using `ruma::RoomId` here and below?